app/controllers/assessments_controller.rb

```diff
@@ -188,10 +188,14 @@ def update
     # Figure out the change
     delta = []
     typed_reported_symptoms.each do |symptom|
-      new_val = symptom.bool_value
-      old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.bool_value
-      if new_val.present? && old_val.present? && new_val != old_val
-        delta << symptom.name + '=' + (new_val ? 'Yes' : 'No')
+      new_val = symptom.value
+      old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.value
+      case symptom.type
+      when 'BoolSymptom'
+        has_changed = old_val != new_val && !old_val.nil? && !new_val.nil?
```

Comment:
Sorry I'm gonna be a stickler but can't this line also just be pulled out of the switch statement now?